message_definitions/v1.0/common.xml

```diff
@@ -2661,11 +2185,14 @@
       <entry value="40" name="MAV_SENSOR_ROTATION_ROLL_90_PITCH_315">
         <description>Roll: 90, Pitch: 315</description>
       </entry>
+      <entry value="41" name="MAV_SENSOR_ROTATION_ROLL_270_YAW_180">
```

Comment:
this conflicts with ardupilot use of 41. I'm pretty sure this is an alias of an existing one
aliasing two to one confuses the auto-orientation code